src/components/data-exploration/differential-expression-tool/DiffExprCompute.jsx

```diff
@@ -22,6 +22,15 @@ const getCellSetKey = (name) => (name?.split('/')[1] || name);
 const getRootKey = (name) => name?.split('/')[0];
 
 const MIN_NUM_CELLS_IN_GROUP = 10;
+const MIN_NUM_SAMPLES_SHOW_ERROR = 2;
+const MIN_NUM_SAMPLES_SHOW_WARNING = 3;
```

Comment:
I would call these e.g. `MIN_NUM_SAMPLES_OMIT_ERROR` or similar or decrement these by one and make the checks below `<=`.

I don't think it makes lexical sense right now as if there are the minimum number of samples to show an error (`MIN_NUM_SAMPLES_SHOW_ERROR`), you don't show the error.

Reply:
Hmm.. yes, I think it's easier to understand.